js/src/components/lineage/NodeView.tsx

```diff
@@ -36,21 +94,44 @@ interface NodeViewProps {
 
 export function NodeView({ node, onCloseNode }: NodeViewProps) {
   const router = useRouter();
+  const [rowCount, setRowCount] = useState<ModelRowCount>();
+  const [isRowCountLoaded, setRowCountIsLoaded] = useState(false);
   const { setSqlQuery } = useRecceQueryContext();
   const withColumns =
     node.resourceType === "model" ||
     node.resourceType === "seed" ||
     node.resourceType === "source";
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { MismatchSummaryModal } = useMismatchSummaryModal();
+  const { icon: resourceTypeIcon } = getIconForResourceType(node.resourceType);
+  const { mutate: queryRowCount } = useMutation({
```

Comment:
This is not mutation, please use useQuery({...., skip: node.resourceType === "model"})